CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.5.0 (October 17, 2020)
```

Comment:
Don't add a changelog entry here because there's another PR queued up. We'll merge both and then publish a new version.